### PR TITLE
bold and italic heading

### PR DESCRIPTION
**Pull request by Pragya2056**
The issues 
#68 
#69 
#70 
are looked at with a single PR.

### CODE REVIEW

**Comment by Pragya2056** [comment]
@YoshithaRathnayake please review it

**Comment by Pragya2056** [comment]
The | Yoshitha is removed.

**Comment by Pragya2056** [comment]
The heading is bold.

**Comment by Pragya2056** [comment]
The heading is italic

File: index.html [hunk 1]
```diff
@@ -53,15 +53,15 @@
 
     <!-- Title -->
 
-    <title>CALCULATOR | YOSHITHA</title>
+    <title>CALCULATOR</title>
```
Comment (YoshithaRathnayake) [inline index.html:56]: This part is not assigned to you therefore undo it @Pragya2056

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -53,15 +53,15 @@
...
 
 </head>
 
 
 <body>
 
 
-  <header class="text-center title">CALCULATOR | YOSHITHA</header>
+  <header class="text-center title" style="font-style: italic bold" >CALCULATOR </header>
```
Comment (YoshithaRathnayake) [inline index.html:64]: remove the italic option bcs italicizing is not an issue assigned to you @Pragya2056

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (YoshithaRathnayake) [inline index.html:64]: Don't use internel css use externel css @Pragya2056

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Pragya2056) [reply index.html:64]: Italics was also assigned to me.

Comment (YoshithaRathnayake) [reply index.html:64]: But @JP109 was done before it @Pragya2056

File: index.html (hunk 1 above)
Comment (Pragya2056) [reply index.html]: This was done.

File: index.html (hunk 2 above)
Comment (Pragya2056) [reply index.html:64]: Sorry I din't realize that.

Comment (YoshithaRathnayake) [reply index.html]: @JP109 did this before you so I merge it so you reverse the change @Pragya2056

Comment (Pragya2056) [reply index.html]: This is resolved.

Comment (Pragya2056) [reply index.html]: I have corrected it.

**Comment by Pragya2056** [comment]
All the requested changes have been made. The conflicts has been resolved. 
@YoshithaRathnayake

File: assets/css/styles.css [hunk 3]
```diff
@@ -1,39 +1,43 @@
...
+
+	#paragraph {
+		margin: 0 auto;
+		font-size: 0.8rem;
+	}
 }
+
+img {
+	width: 20%;
+}
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:43]: Add a new line at the bottom of the page @Pragya2056

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by Pragya2056** [comment]
@YoshithaRathnayake 
Done

**Review by YoshithaRathnayake — APPROVED** [review]
Good Job !
Thank You @Pragya2056

**Pull request merged**